package.json

```diff
@@ -19,58 +19,54 @@
     "@sentry/integrations": "^5.29.1",
     "axios": "^0.21.1",
     "babel-plugin-transform-decorators": "^6.24.1",
-    "core-js": "^3.7.0",
+    "core-js": "^3.8.0",
     "http-status-codes": "^1.4.0",
     "launchdarkly-js-client-sdk": "^2.18.3",
     "lodash": "^4.17.20",
-    "moment": "^2.27.0",
+    "moment": "^2.29.1",
+    "moment-timezone": "^0.5.33",
```

Comment:
I'm going to see about using `public/js/date-time-format-timezone-complete-min.js` instead of Moment. (On my long-term todo list.)